node/src/types/block.rs

```diff
@@ -357,11 +371,13 @@ impl BlockPayload {
         let accusations = (0..num_accusations)
             .map(|_| PublicKey::random(rng))
             .collect();
+        let past_finality_signatures = Default::default();
```

Comment:
I'd prefer if we actually generated something random here - like, create some random number of validators' public keys, then choose a random sample of them and create `PastFinalitySignatures` based on those values.

Reply:
Hmm, good point. I was thinking it might matter for some tests, but we don't save the keys anywhere, anyway, so random bytes should be just as good :+1:

Reply:
Done